code/game/machinery/doors/airlock.dm

```diff
@@ -309,6 +309,8 @@
 	qdel(note)
 	for(var/datum/atom_hud/data/diagnostic/diag_hud in GLOB.huds)
 		diag_hud.remove_from_hud(src)
+	if(brace) //yogs
```

Comment:
If I was acoustic I'd probably tell you to move this to its own file, but it's fine

Reply:
and i am acoustic, so go make a whole .dm with 10 lines just to this shit and ..()